src/Components/Screen.js

```diff
@@ -0,0 +1,7 @@
+const Screen = () => (
+  <div className="screen">
+    0
+  </div>
+);
```

Comment:
- [ ] Kindly convert the `Screen` component to `class-based` instead of `function-based` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).). All components should be class-based.